.gitignore

```diff
@@ -68,7 +70,7 @@ classes/
 .local/share/JetBrains/
 .java/
 out/
-
+/src/main/resources/config/application-artemis.yml
```

Comment:
Please remove them from here and use a local setup to prevent staging these files.

Reply:
i removed these paths/files from the gitignore